cvat/apps/engine/cache.py

```diff
@@ -65,63 +70,166 @@
 
 
 class MediaCache:
-    def __init__(self) -> None:
-        self._cache = caches["media"]
-
-    def _get_checksum(self, value: bytes) -> int:
+    _QUEUE_NAME = settings.CVAT_QUEUES.CHUNKS.value
+    _QUEUE_JOB_PREFIX_TASK = "chunks:prepare-item-"
+    _SLEEP_TIMEOUT = 0.2
+    _CHUNK_CREATE_TIMEOUT = 50
+    _CACHE_NAME = "media"
+    _RQ_JOB_RESULT_TTL = 60
+    _RQ_JOB_FAILURE_TTL = 3600 * 24 * 14 # 2 weeks
+    _PREVIEW_TTL = 3600 * 24 * 7
```

Comment:
Consider making some of this configurable via env variables. For instance, timeouts and intervals are good candidates for this.